index.html

```diff
@@ -69,7 +74,8 @@ <h1 class="first-text">I’m Alzubair Glad to see you!</h1>
     </div>
 
     <section class="section-3">
-      <div class="card">
+      <!-- here -->
+      <div class="card" id="cards">
         <div class="photo">
           <img
             class="work-img"
```

Comment:
- [ ] Kindly make sure you fix the styling issue on your project's cards in mobile size as they no longer look like on Figma design. 
##### Here is how they are looking now:
![mis](https://user-images.githubusercontent.com/83042742/173698447-b5e08a7c-96ea-48a4-88c9-969777d4922e.png)